cra-tf-validate-ignore-goals.json

```diff
@@ -1,3 +1,34 @@
 {
-    "scc_goals": []
+    "scc_goals": [
```

Comment:
The `complete` plan takes a few hours to provision I think because we are using the `enterprise-3nodes-2tb` plan. We could change the plan to use a more lightweight instance?

Reply:
yea, lets use `standard` in the complete example. We may use the `enterprise-3nodes-2tb` in the upcoming FSCloud profile

Reply:
Updated to use complete example which requires 1 ignore to pass cra.